File: HSG_YEN_BAI/contestants/khoi/FROG.py
import math 
with open ("FROG.inp","r") as f:
     n=int(f.readline())
     k=int(f.readline())
     f.close()
with open("FROG.out","w") as f:
    f.write(str(math.gcd(n,k)))
    f.close()

File: HSG_YEN_BAI/contestants/hung/frog.cpp
#include <bits/stdc++.h>
using namespace std;
int main()
{
    ifstream inp("frog.inp");
    ofstream out("frog.out");
    long long n,k;
    inp >> k >> n;
    out << __gcd(n,k);
    out.close();
    return 0;
}
